### ISSUE TITLE
Support lightweight mode

### ISSUE THREAD

**Issue by fresheneesz** [etotheipi/BitcoinArmory#332]
I don't want to have to download the entire blockchain these days. But Armory is the only client I've found that takes security seriously. I can't use armory because it takes DAYS to DL the blockchain before I can use it, and I can't use any other system because they're fucking amateurs. Armory lightweight mode please!

Look how multibit ignores their security fail: https://github.com/keepkey/multibit-hd/issues/952#issuecomment-279870625

**Comment by achow101** [comment]
First of all, Armory is no longer developed on this repo but rather on https://github.com/goatpig/BitcoinArmory.

Secondly, you cannot get security and privacy in addition to convenience, they are mutually exclusive. While lightweight wallets are convenient, you are risking both your security and privacy because you are relying on someone else to validate transactions and blocks and to supply with the correct data. Furthermore, you will be revealing all of the addresses that you use to whoever you are connected to. There are tradeoffs, and if you want the security and privacy that Armory provides, you won't be getting any convenience for it.

That being said, litenode is something that is being worked on. It is fairly similar to Electrum wherein you have servers that are full nodes that provide data to a remote wallet. Unfortunately, this means that you are going to be trading off security and privacy for that convenience. Furthermore, this feature isn't fully complete yet and there are no public servers like Electrum does. You would need to set up your own server and have enough technical knowhow to configure Armory to connect to said server.

**Comment by fresheneesz** [comment]
I'm ok with trading *some* security and privacy (minimal in my mind) for lots of convenience. I trust the Armory devs to treat security with the respect it deserves and maximize it for a lightweight wallet. I've looked into a lot of other wallets and have been disappointed in what I've found. I haven't found anyone else I trust. 

"Armory is no longer developed on this repo"

Thanks for the note. Why does this repo not have a deprecated note?

**Comment by achow101** [comment]
> Thanks for the note. Why does this repo not have a deprecated note?

Because the versions after 0.93.3 are technically a fork of Armory (forked by one of Armory's developers after the company behind Armory stopped developing it and basically ceased to exist). This repo is thus unmaintained and the current developer of Armory does not have access to this repo to do anything with it. This repo is the repo of the "official" version of Armory, but the defacto official version is maintained by goatpig ("official" being the last version released by ATI, the company behind Armory).

tl;dr It's really complicated. Just know that anything you submit to this repo or try to do with this repo basically won't be looked at or acted upon.

**Comment by fresheneesz** [comment]
Gotcha, thanks!
